spring-integration-core/src/main/java/org/springframework/integration/expression/ExpressionEvalMap.java

```diff
@@ -51,8 +51,8 @@
  *	.usingCallback(new EvaluationCallback() {
  *		Object evaluate(Expression expression) {
  *			// return some expression evaluation
- *		}
- *	})
+ *        }
+ *    })
  *	.build();
```

Comment:
Change from tabs to spaces has messed up alignment.

Reply:
Doesn't look like, but I've changed everything into spaces. I don't think tabs are good for JavaDocs.

Reply:
Right, but you had mixed tabs and spaces here. The indentation is still wrong (extra space before `}`). Will fix on merge.